package.json

```diff
@@ -45,35 +46,38 @@
     "rlp": "2.2.3",
     "solidity-coverage": "0.5.11",
     "solparse": "2.2.5",
-    "truffle": "5.0.19",
+    "truffle": "beta",
     "ts-node": "8.1.0",
-    "typescript": "3.4.5",
+    "typescript": "3.5.3",
     "wait-port": "0.2.2",
     "web3": "1.0.0-beta.37",
     "ts-generator": "0.0.8",
-    "typechain": "0.3.14"
+    "typechain": "0.3.14",
+    "fs-extra": "7.0.1"
   },
   "scripts": {
     "update": "git submodule update --init --recursive && npm ci",
     "compile": "truffle compile",
-    "compile:ts": "tsc --target es5",
-    "compile:all": "truffle compile --all && tsc --target es5",
-    "generate_test_proofs": "./proof_generation/main.sh",
+    "compile:ts": "tsc --resolveJsonModule",
```

Comment:
--resolveJsonModule should not be needed in my understanding as in tsconfig.json we've set it to true.

Reply:
Removed this.